libraries/AP_Motors/AP_MotorsUGV.h

```diff
@@ -39,7 +42,7 @@ class AP_MotorsUGV {
     };
 
     // initialise motors
-    void init();
+    void init(uint8_t ftype);
```

Comment:
let's name the argument "frame_type" and ideally we should use the enum

Reply:
Can't do that because it conflicts with the enum name. Also I did originally have this as enum but that requires the caller use a enum which they are not doing.